src/views/v2/components/messages.php

```diff
@@ -20,16 +20,17 @@
 	return;
 }
 
-// @todo @juanfra Make this beautiful.
+$classes = [ 'tribe-events-c-messages', 'tribe-common-b2' ];
+
 ?>
 
-<div class="tribe-events-notices" style="text-align: center; width: 100%; margin: 0 0 1em auto;">
-	<?php
-	foreach ( $messages as $message_type => $message_group ) {
-		foreach ( $message_group as $message ) {
-			echo wp_kses_post( "{$message_type}: $message" );
-		}
-	}
-	?>
+<div <?php tribe_classes( $classes ); ?>>
+	<?php foreach ( $messages as $message_type => $message_group ) : ?>
```

Comment:
Semantically maybe we should updated this to be a `<ul>` & set of `<li>`s, thoughts?